### PR TITLE
os/kola: send summary of failures to slack

### PR DESCRIPTION
**Pull request by arithx**
If any tests fail in a kola run posts a slack message with a list of
failed tests. Parses the list of failed test from the json output via
jq.

### CODE REVIEW

**Comment by arithx** [comment]
Do we want to add the `BUILD_ID` to the slack messages and/or a link?

**Comment by ajeddeloh** [comment]
Looks like you added the link, yes? It would also be pretty slick to add a link to the `_kola_temp.tar.xz` as well.

**Comment by arithx** [comment]
@ajeddeloh I did add the `BUILD_URL`, I'll look at ways of constructing a link to the `_kola_temp.tar.xz`

**Comment by ajeddeloh** [comment]
I take it take points to the job status page (e.g. `https://jenkins-url.example/job/os/job/kola/job/packet/443/`)

**Comment by arithx** [comment]
Yes (according to their documentation) https://wiki.jenkins.io/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-belowJenkinsSetEnvironmentVariables

**Comment by arithx** [comment]
@ajeddeloh updated to include a link to both the URL & the `_kola_temp` (and shortened them so the line is readable instead of just a massive blob of hyperlinks before the failed tests)

**Comment by arithx** [comment]
Fixed link, dropped extraneous strings & compacted the lines.

File: os/kola/aws.groovy [hunk 1]
```diff
@@ -68,6 +68,8 @@ timeout --signal=SIGQUIT 60m bin/kola run \
     --tapfile="${JOB_NAME##*/}.tap" \
     --torcx-manifest=torcx_manifest.json
 '''  /* Editor quote safety: ' */
+
+                message = sh returnStdout: true, script: '''jq '.tests[] | select(.result == "FAIL") | .name' -r < _kola_tmp/aws-latest/reports/report.json | paste -sd "," -'''
```
Comment (bgilbert) [inline os/kola/aws.groovy]: s/tmp/temp/

Comment (arithx) [reply os/kola/aws.groovy]: Good catch my brain still loves to make that mistake constantly.

**Pull request merged**